### ISSUE TITLE
[Screen Reader] The Screen Reader Mode toggle missing in JavaScript and Python editors

### ISSUE THREAD

**Issue by Nina-1210** [microsoft/pxt-microbit#6950]
**Describe the bug**
When Windows 11 Narrator is enabled, Narrator continues to read and interact within the JavaScript and Python editors. However, the **Screen Reader Mode** toggle is not available in these editors under Settings.

**Steps to reproduce the behavior:**
1. Enable **Windows 11 Narrator**.
2. Navigate to [Microsoft MakeCode for micro:bit](https://makecode.microbit.org/beta).
3. Open any existing project.
4. Switch to the JavaScript editor.
5. Open **Settings**.
6. Check for **Screen Reader Mode** (under Green Screen).
7. Repeat steps 4–6 in the Python editor.

**Actual behavior**
Screen Reader Mode toggle is missing in both JavaScript and Python editors.
 
**Expected behavior**
Screen Reader Mode toggle should be available in both JavaScript and Python editors, consistent with Blocks editor.

**Screen Recording**
https://github.com/user-attachments/assets/e5f49199-3c40-4e21-a5b1-6e9e87a180aa

**Additional context**
1.OS: Windows 11 25H2
2.Browser: Chrome, Edge, and Firefox
3.makecode.microbit.org version: 8.1.25
4.Microsoft MakeCode version: 12.3.20

**Comment by microbit-matt-hillsdon** [comment]
The two behaviour differences controlled by this mode only affect the blocks editor, so we followed the previous pattern of the "Keyboard Controls" and only show the setting on the blocks editor.

See https://github.com/microsoft/pxt-microbit/issues/6949#issuecomment-4779439157

**Comment by abchatra** [comment]
As designed.
